### DISCUSSION TITLE
Become a Start9 Service Registry (Community Registry)

### DISCUSSION THREAD

**Discussion by jasonpearce** [runtipi/runtipi-appstore#2743] [Ideas]
If creating your own operating system like StartOS is too complicated, another option would be to make Tipi's impressive 200+ app store compatible with StartOS. 

Start9 Marketplace supports adding and accessing of custom registries for maximum resiliency and decentralization.

- https://docs.start9.com/0.3.5.x/user-manual/alt-registries
- https://blog.start9.com/start9-marketplace-strategy/

If you adopt this method, end users get the best of both worlds. They get a hardened Linux OS from Start9 and they get a Tipi app store of 200+ Docker containers they could run on their Start9 node.

**Comment by nicotsx** [comment]
Hello @jasonpearce, thanks for opening a discussion. This seems like a good idea but what would be the point for us to create this start9 registry? Users of start9 can migrate our store easily if they want to

**Reply by jasonpearce** [reply]
Users would get the best of both worlds:

1. You get to leverage a hardened OS for Docker apps that users don't have to build or maintain. Start9 will remain responsible for StartOS.
2. You increase your project and app store exposure to the StartOS users and node runners. 

Lastly, users of Start9 cannot install RunTipi on a StartOS. StartOS is a hardened OS designed to run Docker. Because of that, users are not provided root access by default. 

Think it over. Both projects have similar objectives (to be self sovereign by running open source software on your own server). I think it could be a win win but fully understand you might think otherwise. Thanks for at least considering this idea.

**Reply by jasonpearce** [reply]
Last point before I leave this alone for you to consider. 

By adding a RunTipi app store registry to the StartOS ecosystem, end users will be able to install apps from your app store registry, Start9's app store registry, and a Community Registry.

There will be some overlap in available apps, but not much. Right now, most of the Start9 and Community Registry apps are bitcoin or bitcoin lightning apps. Very few of RunTipi apps are bitcoin focused (if any).

End users will thus benefit from a wider selection of Docker apps on a single and hardened Linux OS.

Thanks again for considering this idea. And thanks for building.

**Comment by jasonpearce** [comment]
Apps in the Start9 app store:
https://marketplace.start9.com/marketplace

Select "Switch to the Community Registry" to review the apps in the only other available app store. My recommendation is to make Tipi an available third app store registry.
